server-side/node-mssql-crud/src/routes/warden.routes.js

```diff
@@ -1,10 +1,11 @@
 import { Router } from "express";
-import { 
-    gatepassApproveOrReject, } from "../controllers/warden.controller"
+import { gatepassApproveOrReject, } from "../controllers/warden.controller"
+import { getApprovedGatepass } from "../controllers/warden.controller"
 
 const router = Router();
 
 router.get('/warden/gatepass_approve_or_reject/:id/:flag', gatepassApproveOrReject);
+router.get('/warden/getApprovedGatepass/:user_id', getApprovedGatepass);
```

Comment:
what is the :user_id parameter for??

Reply:
for getting the particular warden's user id